src/proxy/ProxyConnectionHandler.cpp

```diff
@@ -0,0 +1,42 @@
+#include "proxy/ProxyConnectionHandler.h"
+
+#include <memory>
+
+namespace wavefront {
+    ProxyConnectionHandler::ProxyConnectionHandler(std::string &hostName, unsigned short port) : hostName(hostName),
```

Comment:
might want to stick to 100 char limit else i have to scroll on github to read your code.